common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -126,6 +126,16 @@ trait CommercialSwitches {
     exposeClientSide = true,
   )
 
+  val ampAmazon = Switch(
+    Commercial,
+    "amp-amazon",
+    "Amp inventory is being auctioned through Amazon",
+    owners = group(Commercial),
+    safeState = Off,
+    sellByDate = never,
+    exposeClientSide = false,
```

Comment:
I think if it's not exposed client-side, we won't be able to read it in AMP…

```suggestion
    exposeClientSide = true,
```

Reply:
I must have misunderstood - I thought we'd said it wasn't necessary to expose client-side.